app/helpers/users_helper.rb

```diff
@@ -4,4 +4,18 @@ def new_or_existing_photo(user)
       user.build_photo
     end
   end
+  
+  def select_options_for_gender
```

Comment:
Define the enum for `gender` in the user model. And use that in dropdown as well.

Reply:
Used Enum as suggested.